---
name: Julien
surname: Tap
date: "`r format(Sys.time(), '%B, %Y')`"
qualifications: PhD
position: Research Scientist
address: France
email: julien.tap@inrae.fr
www: fme.micalis.fr
profilepic: 20210704154818_IMG_7493_nb.jpg
aboutme: After several internships in microbiology at Institut Pasteur (Patrick Grimont, Philippe Glaser, Sylvain Brisse), Julien Tap went to Joel Doré’s Lab to do a PhD headed by Marion Leclerc (Jan 2007 to Dec 2009). His main project was to study dietary fiber impact on human gut microbiota using notably metagenomic and metatranscriptomic approaches. Since his PhD, his research interest focused on interactions between nutrition, gut microbiota, and human health. This allowed him to work with different AP-HP clinicians like Pr Karine Clément and Pr Iradj Sobhani. As post-doc did at EMBL in Peer Bork’s group and at INRAE Metagenopolis unit (Joel Doré, Dusko Ehrlich), as well as Danone research, he used several approaches like numerical ecology and machine learning technics to untangle big omics data with clinical and dietary data. Julien is currently a research scientist at INRAE Micalis institute settled in Paris-Saclay campus, where he works on food and gut microbiome science topics around nutrition and human health.
github: tapj
linkedin: julientap
twitter: julientap
headcolor: "1E90FF"
output: 
  vitae::awesomecv:
    keep_tex: true
    latex_engine: xelatex
---

```{r setup, include=FALSE}
library(tidyverse)
library(vitae)
knitr::opts_chunk$set(echo = FALSE)
```

# Research experience

## Career/Academic appointment
```{r experience}
tribble(
  ~ role, ~ company, ~ loc, ~ dates, ~ details,
  
  "Research scientist", "INRAE Micalis", "Jouy-en-Josas", "2022 - Present", NULL,
  
  "Microbiome scientist", "Danone Nutricia Research", "Paris-Saclay, France", "2015 - 2022", 
    list("Ensure high quality science, deliverables and publications",
         "Manage external collaborations"),
  
  "Postdoctoral Researcher", "INRA Metagenopolis", "Jouy-en-Josas, France", "2013 - 2014", NULL,
  
   "Postdoctoral Researcher", "AP-HP CHU Henri Mondor", "Créteil, France", "2012 - 2013", NULL,
  
  "Postdoctoral Researcher", "EMBL Peer Bork group Computational Biology", "Heidelberg, Germany", "2010 - 2012", NULL,
  
  "PhD student", "INRA Doré Lab", "Jouy-en-Josas, France", "2007 - 2009", "Thesis project : Diet impact on structural and functional dynamics of the human intestinal microbiota",
  
  "Intership student","Institut Pasteur", "Paris, France", "2004-2006", list(
    "Master’s degree project: genetic structure of M. prototuberculosis and origin of M. tuberculosis (S. Brisse)",
    "Master’s degree project : Development of a biodiversity DNA chips for molecular typing of Listeria (P.Glaser)",
    "Bachelor’s degree project : molecular characterization of E. coli O111 and the diversity of isolated strains in France (F.Grimont)"
  )
) %>% 
  detailed_entries(role, dates, company, loc, details)
```
## Consulting and Science outreach 

```{r outreach}

tribble(
  ~ activity, ~ loc, ~ dates, ~ details,

"Industrial Metagenomics Course", "Orsay, France", "2018-2019", "Master student teaching about Metagenomics application in dairy food industry",

"IHMC Next-Gen Scientist Symposium", "Barcelona, Spain", "2018", "co-organizer and communication outreach",

"IHMC Next-Gen Scientist Symposium", "Houston, Texas, USA", "2016", "co-organizer and communication outreach",

"Paris Metagenomics Analysis Group", "Paris, France", "2015", "co-organizer and chairman",

"Gut Microbiota for Health - Web editor", "Paris, France", "2010-2014", "Web Editor, Science curation and Blog post writing"

) %>%
  
   brief_entries(glue::glue("\\textbf{<activity>}: <loc> \\newline <details>", .open = "<", .close = ">"), paste0(dates, "\\newline"), .protect = FALSE)
  
  
  


```
\pagebreak

## University Tutoring

```{r tutoring2}

tribble(
  ~ activity, ~ loc, ~ dates, ~ details,
  
  "Ruben Ladeira", "Master genie Biologie polytech Nice sophia", "2022", "Structural and functional variations of gut Bifidobacterium metagenomes are associated with human gut microbiome alterations and health status",
  
  "Iñigo Oyarzun Lafuente", "Universitat Pompeu Fabra Barcelona - Master Bioinformatics for Health Sciences", "2021", "Changes in gut microbiota metatranscriptome induced by a Bifidobacterium lactis CNCM I-2494 fermented milk are associated with improved gastrointestinal comfort",
  
  "Agnes Barnabé", "University of Paris-Saclay - M2 Biologie Computationnelle", "2019", "Détection et quantification de souches issues de produits laitiers fermentés dans le microbiote intestinal à partir de données métagénomiques",
  
  "Houssem Gharbi", "University of Paris-Saclay - M2 Biologie Computationnelle", "2018", "Détection de souches bactériennes dans le microbiote intestinal humain à partir de données de séquences métagénomiques") %>%
  
#   brief_entries(glue::glue("\\textbf{<student>}: <university> \\newline <details>", .open = "<", .close = ">"), paste0(dates, "\\newline"), .protect = FALSE)
  
     brief_entries(glue::glue("\\textbf{<activity>}: <loc> \\newline <details>", .open = "<", .close = ">"), paste0(dates, "\\newline"), .protect = FALSE)
  
  
  
```



# Education

```{r education}
tribble(
  ~ degree, ~ uni, ~ loc, ~ dates, ~ details,
  
  "University degree in sustainable health food and micronutrition", "University of Bourgogne","Dijon, France", "2021-2022", NULL,
  
  "Ph.D. in Physiology and Pathophysiology", "University of Pierre & Marie Curie", "Paris, France", "2007-2009", NULL,
    
  "Master’s degree in Molecular and Cellular Biology", "University of Pierre & Marie Curie", "Paris, France", "2005-2006", 
    NULL,
  "Bachelor’s degree in Biological engineering - Biochemical and Food specialty", "University of Paris-Est Creteil", "Creteil, France", "2001-2004", 
    NULL,
  
         
) %>% 
  detailed_entries(degree, dates, uni, loc, details)
```


# Awards and distinctions

```{r award}

tribble(
  ~ award, ~ org, ~ dates, ~ details, 
  
  "Highly Cited Researchers", "Clarivate Web of Science", "2020","Recognizing the true pioneers in their fields over the last decade, demonstrated by the production of multiple highly-cited papers that rank in the top 1\\% by citations for field and year in the Web of Science",
  
  "Early stage Investigator Recognition", "The American Gastroenterological Association", "2015", "Digestive Disease Week, Washington, 2015",
  
  "Travel Grant fellowship", "JFHOD", "2011", "Société Nationale Française de Gastro-Entérologie") %>%
  
  brief_entries(glue::glue("\\textbf{<award>}: <org> \\newline <details>", .open = "<", .close = ">"), paste0(dates, "\\newline"), .protect = FALSE)
  
  
  
  


```
\pagebreak
# Publications


```{r publications, eval=FALSE, message=FALSE, warning=FALSE, include=FALSE}

scholar_id <- "aYTQ-WkAAAAJ"
library(dplyr)
library(stringr)
library(scholar)
pubs <- scholar::get_publications(id = scholar_id) %>% mutate(author = as.character(author))
pubs %>% 
   # rowwise() %>% 
   # mutate(
   #   author = case_when(
   #     str_ends(author, "\\.\\.\\.") ~ get_complete_authors(id = scholar_id, pubid = pubid),
   #     TRUE ~ author
   #   )
   # ) %>%
distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>%
  arrange(desc(year)) %>%
  filter(year > 2008) %>%
  filter(journaltitle != "bioRxiv") %>%
  RefManageR::as.BibEntry() %>% #knitr::kable()
  RefManageR::WriteBib("tap.bib")





# scholar::get_publications("aYTQ-WkAAAAJ&") %>% 
#   mutate(author = as.character(author)) %>%
#   rowwise() %>% 
#   mutate(
#     author = case_when(
#       str_ends(author, "\\.\\.\\.") ~ scholar::get_complete_authors(id = scholar_id, pubid = pubid),
#       TRUE ~ author
#     )
#   ) %>%
#   #filter(cites > 30) %>%
#   distinct(title, .keep_all = TRUE) %>%
#   transmute(bibtype = "Article", author = as.character(author),
#             title = as.character(title),
#             journaltitle = as.character(journal), year, key = row_number()) %>%
#   arrange(desc(year)) %>%
#   filter(year > 2008) %>%
#   filter(journaltitle != "bioRxiv") %>%
#   RefManageR::as.BibEntry() %>%
#   RefManageR::WriteBib("tap.bib")

bibliography_entries("tap.bib") %>%
arrange(desc(issued)) 



```


## Research Articles

```{r message=FALSE, warning=FALSE}
library(rvest)
source("getCompleteAuthors.R")

```



```{r publications2, message=FALSE, warning=FALSE}
conference_preceding = c("MICROBIAL ECOLOGY","Microbial Ecology","Obesity Reviews", "Diabetes & Metabolism",
                         "Arthritis & Rheumatology", "The Second International Conference on Holobionts",
                         "Current Developments in Nutrition","United European Gastroenterology Week",
                         "International Human Microbiome Consortium Congress","Digestive Disease Week",
                         "Journées Francophones d'Hépato-gastroentérologie et d'Oncologie Digestive",
                         "ARTHRITIS & RHEUMATOLOGY","Conference on Gastrointestinal Function", 
                         "Metabolomics & Fermented Foods", "NutrEvent")

articles_to_remove = c("bioRxiv","MICROBIAL ECOLOGY","Université Pierre et Marie Curie",
                      "Microbial Ecology","Obesity Reviews", "Diabetes & Metabolism",
                      "EP Patent EP", "Arthritis & Rheumatology", "Current Opinion in Food Science",
                      "médecine/sciences", "The Second International Conference on Holobionts",
                      "Reference Module in Food Science", "Current Developments in Nutrition", 
                      "Colloque Microbiote: les micro-organismes dans la vie quotidienne",
                      "Webinar MetaGenoPolis","NEUROGASTROENTEROLOGY AND MOTILITY",
                      "International Human Microbiome Consortium Congress",
                      "Journées Francophones d'Hépato-gastroentérologie et d'Oncologie Digestive",
                      "United European Gastroenterology Week","Paris Metagenomics Analysis Group","Conference on Gastrointestinal Function",conference_preceding)



reviews_book = c("Current Opinion in Food Science","médecine/sciences","Reference Module in Food Science")

patent_id = "EP Patent EP"



scholar_id <- "aYTQ-WkAAAAJ"
library(dplyr)
library(stringr)
library(scholar)
pubs <- scholar::get_publications(id = scholar_id) %>% mutate(author = as.character(author))
pubs %>% 
      rowwise() %>% 
      mutate(
        author = case_when(
          str_ends(as.character(author), "\\.\\.\\.") ~ get_complete_authors(id = scholar_id, pubid = pubid),
          TRUE ~ as.character(author)
      )
      ) -> pubs



```


```{r publications3, message=FALSE, warning=FALSE}
pubs %>%
   ungroup() %>%
distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>%
  arrange(desc(year)) %>%
  filter(year > 2008) %>%
  filter(!(journaltitle %in% articles_to_remove)) %>%
  filter(!(grepl("Nutrients 2021",title))) %>%
  filter(!(grepl("Mo1942-",title))) %>%
  # rbind(
  #   
  #   tribble(
  # ~ bibtype, ~ author, ~ title, ~ journaltitle, ~ year, ~ key,
  # "Article", "T Vanhaecke, O Bretin, M Poirel, J Tap", "Drinking water source and intake are associated with distinct gut microbiota signatures", "The Journal of Nutrition", "2021", 99
  # )
  # ) %>%
  
  arrange(desc(year)) %>%
  mutate(key = row_number()) %>%
  detailed_entries(what = author,when = year, where = journaltitle, with = paste0(key,". ", title)  )
  #brief_entries(what = paste0(author,". ",title), when = year, with= journaltitle)
  
  #brief_entries(glue::glue("\\textbf{<title>}: <journaltitle> \\newline <author>", .open = "<", .close = ">"), paste0(year, "\\newline"), .protect = FALSE) #%>%
  #mutate(what = what %>% as.character)# %>% gsub("J Tap",'\\textbf{J Tap}'))
```



\pagebreak
## Reviews and books chapter

```{r reviews}

pubs  %>%
   ungroup() %>%
distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>%
  arrange(desc(year)) %>%
  filter(year > 2008) %>%
  filter((journaltitle %in% reviews_book)) %>%
  
  rbind(
    
    tribble(
  ~ bibtype, ~ author, ~ title, ~ journaltitle, ~ year, ~ key,
  "Article", "J Doré, J Tap, D Ehrlich", "Techniques d'identification et d'exploration du microbiote", "John Libbey Eurotext", "2017", 99,
    "Article", "J Doré, J Tap, D Ehrlich", "Composition du microbiote intestinal", "John Libbey Eurotext", "2017", 99
  )
    
    
  ) %>%
  arrange(desc(year)) %>%
  mutate(key = row_number()) %>%
  detailed_entries(what = author,when = year, where = journaltitle, with = paste0(key,". ", title)  )


```


## Patent

```{r patent}

pubs  %>%
   ungroup() %>%
distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>%
  arrange(desc(year)) %>%
  filter(year > 2008) %>%
  filter((journaltitle %in% patent_id)) %>%
  
  mutate(key = row_number()) %>%
  detailed_entries(what = author,when = year, where = journaltitle, with = paste0(key,". ", title)  )


```


## Conference preceedings

```{r preceedings}

pubs %>%
   ungroup() %>%
distinct(title, .keep_all = TRUE) %>%
  transmute(bibtype = "Article", author = as.character(author),
            title = as.character(title),
            journaltitle = as.character(journal), year, key = row_number()) %>%
  arrange(desc(year)) %>%
  filter(year > 2008) %>%
  filter((journaltitle %in% conference_preceding)) %>%
  
  mutate(key = row_number()) %>%
  detailed_entries(what = author,when = year, where = journaltitle, with = paste0(key,". ", title)  )



```





